torchtune/_cli/download.py

```diff
@@ -159,3 +202,91 @@ def _download_cmd(self, args: argparse.Namespace) -> None:
             *list(Path(true_output_dir).iterdir()),
             sep="\n",
         )
+
+    def _download_from_kaggle(self, args: argparse.Namespace) -> None:
+        """Downloads a model from the Kaggle Model Hub."""
+
+        # Note: Kaggle doesn't actually use the "repository" terminology, but we still reuse args.repo_id here for simplicity
+        model_handle = args.repo_id
+        self._validate_kaggle_model_handle(model_handle)
+
+        # Default output_dir is `/tmp/<model_name>`, stripping the publisher name to align with HF output_dir format
+        output_dir = args.output_dir
+        if output_dir is None:
+            model_name = "/".join(args.repo_id.split("/")[1:]).lower()
```

Comment:
See other comment on about using the handle parse function in kagglehub.

Reply:
I forgot that hugging face model names include all the variation info as well. For simplicity, I just opted to make this the full model handle instead of just the model name